src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java

```diff
@@ -177,6 +180,11 @@ public Builder setRetryBackoffMs(long retryBackoffMs) {
       return this;
     }
 
+    public Builder setBehaviorOnNullValues(DataConverter.BehaviorOnNullValues behaviorOnNullValues) {
+      this.behaviorOnNullValues = behaviorOnNullValues;
```

Comment:
Maybe just to be safe:
```
this.behaviorOnNullValues = behaviorOnNullValues != null ? behaviorOnNullValues : BehaviorOnNullValues.DEFAULT
```

Reply:
Is implicit behavior a good idea here? Seems like throwing an NPE might help avoid confusion down the road--if `null` is intentionally being passed, a suggestion could be included in the exception message to instead pass `DEFAULT` explicitly; if `null` is accidentally being passed, it could save some headache over unexpected behavior.

Reply:
Sure, disallowing the parameter to be null is perfectly acceptable. This is internal code that will never be null, and in this case I'd be fine with an assert to document that null is not allowed and to prevent changes in the future that might pass null. OTOH, if this were public code or even code that was more difficult to control, then we'd actually want to check the parameter and throw an IllegalArgumentException.

Reply:
Alright, will add a call to `Objects.requireNonNull()` here and (hopefully) adequate documentation to help anyone encountering the resulting NPE.